web/src/views/User/UserDetailPage.vue

```diff
@@ -0,0 +1,191 @@
+      }
+    }
+    const isTokenVisible = ref(false);
+    const isCopyRefreshTokenSnackbarVisible = ref(false);
+
+    const handleRefreshTokenCopyButtonClick = async () => {
+      if (refreshToken != null) {
+        await navigator.clipboard.writeText(refreshToken);
+        isCopyRefreshTokenSnackbarVisible.value = true;
+      }
+    };
+
+    const handleRefreshTokenVisibilityButtonClick = () => {
+      isTokenVisible.value = !isTokenVisible.value;
+    };
+
+    const handleRefreshTokenInputClick = (event: MouseEvent) => {
+      event.preventDefault();
+      if (isTokenVisible.value) {
+        (event.target as HTMLInputElement).select();
+      }
+    };
+
+    return {
+      user: stateRefs.user,
+      userLoading: stateRefs.userLoading,
+      origin: window.location.origin,
+      refreshToken,
+      refreshTokenExpirationDate,
+      isCopyRefreshTokenSnackbarVisible,
+      isTokenVisible,
+      handleRefreshTokenVisibilityButtonClick,
+      handleRefreshTokenInputClick,
+      handleRefreshTokenCopyButtonClick,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-main>
+    <AppBanner v-if="false" />
+    <v-container>
+      <div v-if="userLoading">
+        Loading...
+      </div>
+      <div v-else-if="user">
+        <div class="mb-8">
+          <div class="py-2">
+            <span class="text-h4">{{ user.name }}</span>
+            <v-chip
+              v-if="user.is_admin"
+              class="mx-2"
+              style="vertical-align: baseline"
+              color="primary"
+            >
+              <v-icon
+                left
+              >
+                mdi-shield-account
+              </v-icon>
+              Admin
+            </v-chip>
+          </div>
+          <orcid-id
+            :orcid-id="user.orcid"
+            :authenticated="false"
+            :width="24"
+          />
+        </div>
+
+        <div class="mb-8">
+          <div class="text-h5 py-2">
+            Developer Tools
+          </div>
+          <p>
+            To access authenticated endpoints in the <a href="/api/docs">NMDC Data and Submission Portal API</a> you
+            must provide an <b>Access Token</b> in the <code>Authorization</code> header of the request. An
+            <b>Access Token</b> is valid for 24 hours and can be obtained by providing your <b>Refresh Token</b> to the
+            <code>/auth/refresh</code> endpoint. Your <b>Refresh Token</b> and its expiration date can be found below.
+            <b>Treat this Refresh Token as securely as you would treat a password!</b>
+          </p>
+
+          <v-row>
+            <v-col
+              cols="auto"
+            >
+              <v-text-field
+                label="Refresh Token"
+                readonly
+                filled
+                :type="isTokenVisible ? 'text' : 'password'"
+                :value="refreshToken"
+                @click="handleRefreshTokenInputClick"
+              >
```

Comment:
Any thoughts on using a [text area component](https://dev.vuetifyjs.com/en/components/textareas/#auto-grow) or making the text area longer? The token is much longer than the current default width of the text field, which might make it difficult for users who miss the "copy to clipboard" button and try to highlight + copy.

Reply:
I'm pretty ambivalent about this one, BTW. In firefox on my OS (Ubuntu) I seemingly can't even highlight a partial bit of the token. On chrome It's easier to see that the token extends beyond the visual textbox (but I can highlight only a piece of it). A tooltip on the "copy to clipboard" button might be nice to have as well.

Reply:
I did think about using a text area. I felt like my choices were 1) use a text input field and toggle the `type` attribute to obscure the token or 2) use a text area field and implement my own method of obscuring (since text areas don't have a `type` attribute). I opted for 1 in an attempt to keep things simpler. 

The `@click` handler on the input itself was an attempt at mitigating the issue that you can't see the whole token, and, worse still, double clicking on the text only selects up to the first period in the token. But as you pointed out maybe that interaction is a little confusing. So I've removed it in the latest changes and instead the entire text gets selected when you click the "reveal token" button.

Tooltips on both icon buttons have been added.